### PR TITLE
Minor improvements

### LINKED ISSUE #978: FAQ Ask Footer
A footer on the FAQ page that invites people to email / submit a question.

### LINKED ISSUE #1001: Paragraphs and header spacing
lbry.com/what highlights issues with paragraphs and header spacing. 

1. If headers are going to have bottom margins, presumably smaller headers should have smaller margins. Right now, `h2` has no margin while `h4` has one. This makes spacing weird.
1. Paragraphs should probably not have a top-margin when following a header or something else should change so that headers are always more closely coupled to the text below.

### PR DESCRIPTION
**Pull request by NetOperatorWibby**
### Description

Improving spacing between header elements and content on ancillary pages and added an email link on the FAQ page.

### Fixes

Closes #978
Closes #1001

### CODE REVIEW

File: view/template/content/faq.php [hunk 1]
```diff
@@ -44,4 +44,10 @@
       <?php endforeach ?>
     </div>
   </section>
+
+  <section>
+    <div class="inner-wrap">
+      <p>Do you have further questions or want to submit a question to our FAQ? <a href=" mailto:growth@lbry.com?subject=Question%20for%20the%20FAQ" rel="noopener noreferrer" target="_blank" title="">Send us an email</a>!</p>
```
Comment (NetOperatorWibby) [inline view/template/content/faq.php]: Is this the correct email address to use?

Comment (NetOperatorWibby) [reply view/template/content/faq.php]: @kauffj ^

Comment (kauffj) [reply view/template/content/faq.php]: @tzarebczan any thoughts or preferences on how to collect these? We could link a Google form or just invite to email.

File: web/scss/pages/_ancillary.scss [hunk 2]
```diff
@@ -8,17 +8,45 @@
...
+    + form,
+    + h1,
+    + h2,
+    + h3,
+    + h4,
+    + h5,
+    + ol,
+    + p,
+    + ul {
+      padding-top: 0.25rem;
```
Comment (kauffj) [inline web/scss/pages/_ancillary.scss]: Why not all margin? You want the margin collapse behavior on the top elements but not the bottom ones?

File: web/scss/pages/_ancillary.scss [hunk 3]
```diff
@@ -8,17 +8,45 @@
...
+    }
+
+    + .button {
+      display: inline-block;
```
Comment (kauffj) [inline web/scss/pages/_ancillary.scss]: This is a bad rule... is this just to make the padding/margin above work?

If this is just to fix visual spacing for a button that immediately follows a header, it seems fine to wrap it in a `<p>` or a `<div>`

File: web/scss/pages/_ancillary.scss (hunk 2 above)
Comment (tzarebczan) [reply web/scss/pages/_ancillary.scss]: Emailing hello@lbry.com will work best.

File: web/scss/pages/_ancillary.scss (hunk 3 above)
Comment (NetOperatorWibby) [reply web/scss/pages/_ancillary.scss]: I don't remember where this button is.

**Pull request merged**